apps/api.nameguard.io/package.json

```diff
@@ -1,29 +0,0 @@
-{
-  "name": "nameguard-api",
-  "private": true,
-  "description": "AWS Lambda handler for NameGuard.",
-  "version": "1.0.0",
-  "license": "MIT",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/namehash/namekit.git",
-    "directory": "apps/api.nameguard.io"
-  },
-  "homepage": "https://github.com/namehash/namekit/tree/main/apps/api.nameguard.io",
-  "keywords": [
-    "ENS",
-    "NameKit",
-    "NameGuard",
-    "NameHash"
-  ],
-  "scripts": {
-    "deploy:prod": "DOCKER_CONFIG=.docker serverless deploy --stage prod",
-    "deploy:staging": "DOCKER_CONFIG=.docker serverless deploy --stage staging"
-  },
-  "dependencies": {
-    "serverless": "3.39.0"
```

Comment:
@BanaSeba suggest undoing the delete of this file. And then navigating to this directory and running `pnpm remove serverless`. We need PNPM to remove this dependency. The lock file in the root of the monorepo should be getting updated in this PR too.

Reply:
Updated pnpm as well